### DISCUSSION TITLE
Ignore Formatting for a Line

### DISCUSSION THREAD

**Discussion by HarshKapadia2** [mvdan/sh#1087] [Q&A]
Hello,

I wanted to enquire if it is possible to ignore formatting for a line (or a few lines) using a comment above the line or through some other way.

Thank you!

**Comment by 5HT2** [comment]
Hi, I would also like to know this. The manpage is not super clear on which syntaxes are valid for this.

**Comment by mvdan** [comment]
This is not supported, nor will it be. If there is a bug in the parsing or formatting of the shell, file the bug.

**Reply by 5HT2** [reply]
Well, unfortunately the bugs all relate to things that boil down to properly supporting zsh syntax, most of which have bugs filed.

Considering that https://github.com/mvdan/sh/issues/120 hasn't been resolved yet, I was hoping to be able to still use it for formatting / syntax checking generally outside of explicitly ignored lines.
